Tribler/Core/Modules/restapi/events_endpoint.py

```diff
@@ -139,6 +142,9 @@ def on_channel_discovered(self, subject, changetype, objectID, *args):
     def on_torrent_discovered(self, subject, changetype, objectID, *args):
         self.write_data({"type": "torrent_discovered", "event": args[0]})
 
+    def on_torrent_removed_from_channel(self, subject, changetype, objectID, *args):
+        self.write_data({"type": "torrent_removed_from_channel", "event": args[0]})
```

Comment:
If you put the infohash in objectID you get the same as on_torrent_finished:
``
 "event": {"infohash": objectID.encode('hex'), "name": args[0]}})
``